src/lang/KclSingleton.ts

```diff
@@ -560,8 +560,7 @@ export class KclManager {
       return
     }
 
-    this.ast = { ...ast }
-    return this.executeAst()
+    return this.executeAst({ ast })
```

Comment:
`executeAst` already sets the ast, so passing it through for it to do, other wise it get's set's twice, which is a problem because I added `_lastAst` in order to compare to the previous ast. (use for comparing manual edits when in sketch mode).